### ISSUE TITLE
Not updated beta client?

### ISSUE THREAD

**Issue by Andrej730** [desktop/desktop#16723]
Maybe I'm missing something but currently Beta client version is 3.2.2-beta2.

![image](https://github.com/desktop/desktop/assets/9417531/0b20f795-8e2d-4142-892c-d67b202defc5)

When non-beta client is now already 3.2.3.

![image](https://github.com/desktop/desktop/assets/9417531/32a74dec-aa8e-43e2-801a-b13ba4b5431f)

Does it mean that non-beta client now has more recent changes than beta? Is it intended?

**Comment by tidy-dev** [comment]
@Andrej730 Thanks for reaching out! The production client is not ahead of beta. We did two back to back production releases. The first one was a dependency update that we isolated from the current beta. The second one was a release from the latest beta. Thus, production and beta are currently the same build. We intended to have a beta build out sooner but it was delayed.  We will have new beta out likely next week with the latest on development. Thank you for following!
